### PR TITLE
4 update screenshots

### LINKED ISSUE #4: Update product screenshots
Update all guide product screenshots & walkthroughs to reflect new UI

- [x] Quickstart
- [x] Introduction
- [x] Mod features
- [x] Mod editor
- [x] Using AnyMod
- [x] Examples

### PR DESCRIPTION
**Pull request by damionvega**
Closes #4

### CODE REVIEW

File: docs/guide/usage.md [hunk 1]
```diff
@@ -36,10 +38,47 @@ then paste the Anymod script once, just before the closing `</head>` tag:
 ```html
 <!-- Anymod -->
 <script id="Anymod-script">
-  (function (m,o,d,u,l,a,r,i,z,e) {
-    u[m]={Project:o,rq:[],Opts:r,ready:function(j){u[m].rq.push(j)}};function j(s){return encodeURIComponent(btoa(s))};z=l.getElementById(m+'-'+a);r=u.location;
-    e=[d+'/page/'+o+'/'+j(r.pathname)+'/'+j(r.host)+'?t='+Date.now(),d];e.map(function(w){i=l.createElement(a);i.defer=1;i.src=w;z.parentNode.insertBefore(i,z);});
-  })('Anymod','XXXXXX','https://cdn.anymod.com/v2',window,document,'script',{});
+  ;(function(m, o, d, u, l, a, r, i, z, e) {
+    u[m] = {
+      Project: o,
```
Comment (tyrw) [inline docs/guide/usage.md]: Looks like Prettier bit you here.  Can you revert this to the compact form?

**Comment by damionvega** [comment]
$100?! Yes I'll do that now. Just save them with the same name as in the markdown files, right?

**Comment by tyrw** [comment]
> $100?! Yes I'll do that now. Just save them with the same name as in the markdown files, right?

Yes the name doesn't matter, as long as it can be paired with where it is in the markdown.  Cloudinary will append a random suffix as well, so it won't be using the exact same URLs.

File: docs/guide/usage.md (hunk 1 above)
Comment (damionvega) [reply docs/guide/usage.md]: Fixed

**Pull request merged**